integration_test/itest/cluster.go

```diff
@@ -42,7 +43,7 @@ const TestUser = "telepresence-test-developer"
 const TestUserAccount = "system:serviceaccount:default:" + TestUser
 
 type Cluster interface {
-	CapturePodLogs(ctx context.Context, app, ns string)
+	CapturePodLogs(ctx context.Context, app, container, ns string)
```

Comment:
Nit: Perhaps change `app` to `labelSelector` or `selector` since that's what it was changed to.